api/models/candidates-model.js

```diff
@@ -0,0 +1,35 @@
+const db = require('../../data/dbConfig');
+
+async function getCandidates() {
+  return db('candidates');
+}
+
+function getCandidatesById(id) {
+  return db('candidates')
+    .where({ id })
+    .first();
+}
+
+async function addCandidate(candidate) {
```

Comment:
either remove async or add a function to perform after insert.
if you were to chain a getCandidateById after the insert, then you would need it to be an async function.